pytorch_object_detection/README.md

```diff
@@ -0,0 +1,2 @@
+# computer_vision
+Test and try CV models
```

Comment:
Some information about experiments? Links to tutorials? Information about author?

Reply:
done?

Reply:
Information about you

Reply:
done